qiskit/circuit/library/__init__.py

```diff
@@ -357,6 +357,11 @@
    :toctree: ../stubs/
    :template: autosummary/class_no_inherited_members.rst
 
+   n_local
+   efficient_su2
+   real_amplitudes
+   pauli_two_design
+   excitation_preserving
```

Comment:
what about the classes `TwoLocal, EvolvedOperatorAnsatz` and `QAOAAnsatz` ? they won't be converted into functions?

Reply:
It will be confusing if some of them will remain classes and not functions, or do you plan to totally deprecate them?

Reply:
Note that the functions here appear in the table with the classes, and not after it (like they do in #13361)

Reply:
QAOA and EvolvedOperatorAnsatz are added in #13361 and TwoLocal is redundant since it's 100% covered by n_local 🙂